btag.py

```diff
@@ -0,0 +1,56 @@
+import os
+import ROOT
+from .CorrectionsCore import *
+from Common.Utilities import WorkingPointsbTag
+import yaml
+
+
+
+class bTagCorrProducer:
+    jsonPath = "/cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/BTV/{}/btagging.json.gz"
+    bTagEff_JsonPath = "Corrections/data/BTV/{}/btagEff_TT.root"
+    initialized = False
+    SFSources = ["btagSFbc", "btagSFlight"]
+    correlation = ["nocorr","correlated", "uncorrelated"]
```

Comment:
same here - don't introduce correlation, only sources

Reply:
The names are: source_up/down_correlated/uncorrelated

Reply:
at the python level only unc source should be used.